server/services/storage/agent_framework_storage_service.test.ts

```diff
@@ -263,7 +265,6 @@ describe('AgentFrameworkStorageService', () => {
             input: 'input',
             response: 'response',
             origin: 'origin',
-            parent_interaction_id: 'parent_interaction_id',
```

Comment:
Shall we remove the `interaction_id` and `conversation_id` too? I think we need to add `message_id` here.